modules/manage/partials/iceberg/use-iceberg-catalogs.adoc

```diff
@@ -0,0 +1,106 @@
+* config_ref:iceberg_rest_catalog_client_secret,true,properties/cluster-properties[`iceberg_rest_catalog_client_secret`]: The secret data to connect to the REST catalog.
+
+For REST catalogs that use self-signed certificates, also configure these properties:
+
+* config_ref:iceberg_rest_catalog_trust_file,true,properties/cluster-properties[`iceberg_rest_catalog_trust_file`]: The path to a file containing a certificate chain to trust for the REST catalog.
+* config_ref:iceberg_rest_catalog_crl_file,true,properties/cluster-properties[`iceberg_rest_catalog_crl_file`]: The path to the certificate revocation list for the specified trust file.
+
+See xref:reference:properties/cluster-properties.adoc[Cluster Configuration Properties] for the full list of cluster properties to configure for a catalog integration.
+
+=== Example REST catalog configuration
+
+For example, if you have Redpanda cluster configuration properties set to connect to a REST catalog:
+
+[,yaml]
+----
+iceberg_catalog_type: rest 
+iceberg_rest_catalog_endpoint: http://catalog-service:8181 
+iceberg_rest_catalog_client_id: <rest-connection-user>
+iceberg_rest_catalog_client_secret: <rest-connection-password>
+----
+
+And you use Apache Spark as a processing engine, configured to use a catalog named `streaming`:
+
+[,spark]
+----
+spark.sql.catalog.streaming = org.apache.iceberg.spark.SparkCatalog
+spark.sql.catalog.streaming.type = rest
+spark.sql.catalog.streaming.uri = http://catalog-service:8181
+# You may need to configure additional properties based on your object storage provider.
+# See https://iceberg.apache.org/docs/latest/spark-configuration/#catalog-configuration and https://spark.apache.org/docs/latest/configuration.html
+# For example, for AWS S3:
+# spark.sql.catalog.streaming.io-impl = org.apache.iceberg.aws.s3.S3FileIO
+# spark.sql.catalog.streaming.warehouse = s3://<bucket-name>/
+# spark.sql.catalog.streaming.s3.endpoint = http://<s3-uri>
+----
+
+NOTE: Redpanda recommends setting credentials in environment variables so Spark can securely access your Iceberg data in object storage. For example, for AWS, use `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY`.
+
+Using Spark SQL, you can query the Iceberg table directly by specifying the catalog name, the namespace, and the table name:
+
+[,sql]
+----
+SELECT * FROM streaming.redpanda.<table-name>;
+----
+
+The Iceberg table name is the name of your Redpanda topic. Redpanda puts the Iceberg table into a namespace called `redpanda`, creating the namespace if necessary. 
+
+The Spark engine can use the REST catalog to automatically discover the topic's Iceberg table. Depending on your processing engine, you may need to also create a table in the engine to point the data lakehouse to the table location in the catalog. For an example, see xref:manage:iceberg/redpanda-topics-iceberg-snowflake-catalog.adoc[].
```

Comment:
@kbatuigas  IIUC the need to explicitly CREATE TABLE and the need to point to the specific table location are a little orthogonal.   In Snowflake, we've sen that you need to create table even when using the REST catalog (Open Catalog) but DONT need to provide the exact table location (its implicit from the catalog integration).  With the object storage catalog type, you ALWAYS DO need to provide the exact table location, regardless of which client/lakehouse you're using.   This statement kind of mixes those two things.  Not sure the best way to communicate this but we should probably clarify

Reply:
Updated